src/app/components/itinerary-details/itinerary-details.component.html

```diff
@@ -35,75 +36,64 @@ <h1 class="flex items-center gap-2 text-2xl font-bold text-white">
         </div>
       </div>
       <p class="mt-2 text-lg">
-        {{
-          activity()?.cost | currency: activity()?.currency : 'symbol-narrow'
-        }}
-
-        @if (baseCurrency$ | async; as baseCurrency) {
+        {{ activity.cost | currency: activity.currency : 'symbol-narrow' }}
+        @if (baseCurrency(); as baseCurrency) {
           @if (
-            activity()?.currency !== baseCurrency &&
-            activity()?.costInBaseCurrency
+            activity.currency !== baseCurrency && activity.costInBaseCurrency
           ) {
             <span class="text-xs text-green-1000">
               ±
               {{
-                activity()?.costInBaseCurrency
+                activity.costInBaseCurrency
                   | currency: baseCurrency : 'symbol-narrow'
               }}
             </span>
           }
         }
       </p>
 
-      <app-map [inputActivity]="activity()"></app-map>
+      <app-map [inputActivity]="activity"></app-map>
 
       <div>
-        <nz-steps [nzCurrent]="3" nzDirection="vertical">
-          <nz-step
-            [nzTitle]="(newStartDate() | date) + ' - ' + (newEndDate() | date)"
-            nzIcon="schedule"></nz-step>
-          <nz-step
-            [nzTitle]="
-              activity()?.duration +
-              ' ' +
-              (activity()?.duration === 1 ? 'day' : 'days')
-            "
-            nzIcon="field-time"
-            class="durationStep"></nz-step>
-        </nz-steps>
-        @if (activity()?.notes) {
+        <app-steps
+          [starDate]="newStartDate()"
+          [endDate]="newEndDate()"
+          [duration]="activity.duration"></app-steps>
+        @if (activity.notes) {
           <p class="mt-2 text-xl font-bold">Notes:</p>
-          <p>{{ activity()?.notes }}</p>
+          <p>{{ activity.notes }}</p>
         }
       </div>
       <div
         class="bottom-0 flex w-full justify-end gap-2 pb-10 md:fixed md:max-w-xl md:px-6">
         <a
           type="button"
           class="btn border-0 bg-green-1000"
-          [routerLink]="['/trip', tripId()]">
+          [routerLink]="['/trip', tripId()]"
+          aria-label="Return to trip">
           Return
         </a>
       </div>
     </div>
     <dialog #confirmModal class="modal">
       <div class="modal-box">
-        <h3 class="text-2xl font-bold">Hold on!</h3>
+        <p class="text-2xl font-bold">Hold on!</p>
         <p class="py-4">
           Are you sure you would like to delete
-          {{ activity()?.title ?? 'this activity' }} from
-          {{ tripToUpdate()?.title ?? 'this trip' }}?
+          {{ activity.title }} from {{ tripToUpdate()?.title ?? 'this trip' }}?
```

Comment:
default?

Reply:
I get this when I add the default:
`The left side of this nullish coalescing operation does not include 'null' or 'undefined' in its type, therefore the '??' operator can be safely removed.`
That is why there was no default. Should I still add it regardless?